Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -578,62 +573,92 @@ private extension SPNoteListViewController {
 //
 private extension SPNoteListViewController {
 
-    func rowActionsForDeletedNote(_ note: Note) -> [UITableViewRowAction] {
-        return [
-            UITableViewRowAction(style: .default, title: ActionTitle.restore, backgroundColor: .simplenoteRestoreActionColor) { (_, _) in
+    func contextActions(for note: Note) -> [UIContextualAction] {
+        if note.deleted {
+            return deletedContextActions(for: note)
+        }
+
+        return regularContextActions(for: note)
+    }
+
+    func deletedContextActions(for note: Note) -> [UIContextualAction] {
+        [
```

Comment:
Maybe a little nitpicky of me, but I'm wondering... is it easy to miss what this is doing given the statement is spread out over multiple lines?  Would adding a `return` keep things clear?

Reply:
Good point, done, thank you!